app.js

```diff
@@ -42,16 +38,9 @@ function operate(operator, a, b) {
 const display = document.querySelector(".display");
 const btns = document.querySelectorAll("button");
 
+// put number selected on the display
 btns.forEach((button) => {
-  button.addEventListener("click", (e) => calculatorDisplay(button));
+  button.addEventListener("click", (e) => {
+    display.innerHTML += button.innerText;
+  });
 });
-
-// Create the functions that populate the display when you click the number buttons…
-// you should be storing the ‘display value’ in a variable somewhere for use in the next step.
-
-// put number selected on the display
-function calculatorDisplay(value) {
-  switch (value) {
-    case "zero":
-  }
-}
```

Comment:
More of a general comment on the commit than on the code: here you're removing code that was added in the previous (incomplete) commit. If those two commits were squashed, all the code that was just added and removed or added and refactored would simply not show up in the git history, which in this case is a good thing. A commit is as the name implies: you're "committing" to the changes that you made. So as I said previously: it's fine to commit a lot while working so that you have save points, but it's a good idea to squash some commits before you push so that the state of the code base on any particular commit makes sense, and you don't add code just to remove it later.